### PR TITLE
DCC Ticketing: Consent Two Screen (EXPOSUREAPP-10482)

### PR DESCRIPTION
**Pull request by AlexanderAlferov**
## Whats new
Dcc Ticketing Consent two screen.

Screenshot 2021-11-30 at 00 25 58

## Jira
[Jira](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10482)
[Story](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10194)

## How to test against mock server
1- scan 
```console 
- VC:             cwa vc gen --pii 42
- TC:             cwa tc gen --pii 42
- RC:             cwa rc gen --pii 42
```

2- scan
```console 
cwa ti gen --seed 42 -o
```

## ToDo
- [x] Double check if used proper value for QR
- [x] Check what we should do with transaction context after results

### CODE REVIEW

File: Corona-Warn-App/src/main/res/values-de/dcc_ticketing_strings.xml [hunk 1]
```diff
@@ -83,4 +83,35 @@
...
+    <!-- XTXT: DCC Ticketing: Consent Two: Test Partner -->
+    <string name="dcc_ticketing_consent_two_test_partner">"Prüfpartner:"</string>
+    <!-- XTXT: DCC Ticketing: Consent Two: Test Partner Default Value -->
+    <string name="dcc_ticketing_consent_two_test_partner_default">"“Betreiber_Validationservice”"</string>
+    <!-- XTXT: DCC Ticketing: Consent Two: Provider -->
+    <string name="dcc_ticketing_consent_two_provider">"Anbieter:"</string>
+    <!-- XTXT: DCC Ticketing: Consent Two: Provider Default Value -->
+    <string name="dcc_ticketing_consent_two_provider_default">"“Lufthansa”"</string>
+    <!-- XHED: DCC Ticketing: Consent Two: Body -->
+    <string name="dcc_ticketing_consent_two_body">"Das ausgewählte Zertifikat und Ihre Buchungsdaten werden nun an den Prüfpartner des Anbieters übermittelt.  Für die Übermittlung Ihrer Daten an den Prüfpartner ist Ihr Einverständnis erforderlich."</string>
```
Comment (KathrinLuecke) [inline Corona-Warn-App/src/main/res/values-de/dcc_ticketing_strings.xml:101]: There's one space too much between the two sentences - please remove.

File: Corona-Warn-App/src/main/res/values-de/dcc_ticketing_strings.xml [hunk 2]
```diff
@@ -83,4 +83,35 @@
...
+    <!-- XHED: DCC Ticketing: Consent Two: Point 1 -->
+    <string name="dcc_ticketing_consent_two_point_1">"Der Prüfpartner überprüft, ob die elektronische Signatur echt und das technische Ablaufdatum des Zertifikats noch nicht erreicht ist. Das technische Ablaufdatum und die übermittelten Zertifikatsinhalte können Sie in der App in der Detailansicht des Zertifikats einsehen."</string>
+    <!-- XHED: DCC Ticketing: Consent Two: Point 2 -->
+    <string name="dcc_ticketing_consent_two_point_2">"Die Zertifikate enthalten sensible Gesundheitsdaten. Erlauben Sie daher nur solchen Anbietern die Überprüfung, die Ihnen bekannt sind und denen Sie vertrauen."</string>
+    <!-- XHED: DCC Ticketing: Consent Two: Point 3 -->
+    <string name="dcc_ticketing_consent_two_point_3">"Für die Datenverarbeitung durch den Anbieter und den Prüfpartner ist das RKI nicht verantwortlich. Lesen Sie bitte deren   Datenschutzhinweise, damit Sie wissen, wozu und wie Ihre Daten verwendet werden."</string>
```
Comment (KathrinLuecke) [inline Corona-Warn-App/src/main/res/values-de/dcc_ticketing_strings.xml:107]: 2 spaces too much between "deren Datenschutzhinweise" (Lesen Sie bitte deren   Datenschutzhinweise,...) - please remove

**Review by KathrinLuecke — CHANGES_REQUESTED** [review]
Please remove superfluous spaces - see comments

**Comment by mtwalli** [comment]
@KathrinLuecke  Pleaase have another look?

**Review by KathrinLuecke — APPROVED** [review]
UA approved

**Review by mtwalli — APPROVED** [review]

**Review by BMItr — APPROVED** [review]

**Pull request merged**